docs/content/kubernetes-deploy.md

```diff
@@ -0,0 +1,50 @@
+# Deployment into cluster using Camel JBang Kubernetes plugin
+
+## Manage deployment lifecycle
+
+There are extensions which allows you to manage your deployments (which are part of an [Extension Pack for Apache Camel](https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack)):
+
+- [OpenShift Toolkit](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector)
+- [Kubernetes](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools)
+
+or you can use a power of Camel CLI described below.
+
+### Follow logs
+
+You can use Camel CLI to obtain logs of current running integration. In terminal window execute:
+
+```shell
+jbang camel@apache/camel kubernetes logs --name=<name>
+```
+
+### Remove deployment
+
+To remove current integration, you can use also Camel CLI. In this case the command is:
+
+```shell
+jbang camel@apache/camel kubernetes delete --name=<name>
+```
+
+#### Troubleshooting
+
+For a latest releases of Camel (4.8.1+) there could be problem when deleting deployments using Camel Jbang CLI, for details you can see reported upstream issue [CAMEL-21388](https://issues.apache.org/jira/browse/CAMEL-21388).
+
+In that case please try with previous version which was working better.
+
+```shell
+jbang -Dcamel.jbang.version=4.8.0 camel@apache/camel kubernetes delete --name=<name>
+```
+
+## How to deploy into local Kubernetes cluster
+
+By default, the deployment aims OpenShift cluster. In case you need to deploy into Kubernetes, there is a small modification needed:
+
+1) open Settings UI in VS Code
```

Comment:
I think we could provide the path in UI: File -> Preferences -> Settings

Reply:
added separately for macOS and linux/windows because the path is a bit different 😱